packages/components/pie-textarea/src/index.ts

```diff
@@ -19,25 +22,48 @@ export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
     static shadowRootOptions = { ...LitElement.shadowRootOptions, delegatesFocus: true };
 
     @property({ type: Boolean, reflect: true })
```

Comment:
do you know why we are reflecting the disabled prop here while we don't in some other places?

Reply:
Good question. I just looked and it seems that in all of the form components (checkbox, switch, text input and button) we are reflecting. I suppose this is to make sure that if something externally changes the disabled state of the component that this can be noticed by the form.